app/presenters/account_reset/pending_presenter.rb

```diff
@@ -19,8 +20,19 @@ def time_remaining_until_granted(now: Time.zone.now)
       )
     end
 
-    def account_reset_deletion_period_hours
-      IdentityConfig.store.account_reset_wait_period_days.days.in_hours.to_i
+    def account_reset_deletion_period
```

Comment:
Should this delegate to (or at least call) `account_reset_deletion_period_interval` from the `AccountResetConcern`?